### ISSUE TITLE
Programs and Sub-Categories have wrong breadcrumbs

### ISSUE THREAD

**Issue by alexschedrov** [ymcatwincities/openy#1388]
For some reason Programs and Sub-Categories have wrong breadcrumbs.
![image](https://user-images.githubusercontent.com/1316234/48762969-e2e92a00-ecb4-11e8-9771-003fb16209c7.png)

**Comment by denisbilchenko** [comment]
Hello @Sanchiz 
Could you please advise, problem is with the long title of node after import(or manually configure)? Or problem is with breadcrumbs and we should use
```
white-space: nowrap; 
overflow: hidden;
text-overflow:ellipsis;
```
for the long title?

Thanks.

**Comment by YMCA-GTC** [comment]
@Sanchiz Can you please let us know your thoughts on @denisbilchenko 's comments above so we can prioritize this one appropriately? Thank you!

**Comment by ddrozdik** [comment]
@denisbilchenko This is not about how to support long titles, the issue is in the demo content that creates this ugly breadcrumb.

**Comment by denisbilchenko** [comment]
In the pull request #1222 were changed yaml files of node session, node class and node activity. Also it was deactivated 'no_stub' option to create stub entities. These entities have ugly title, because it was generated. Also they are covered by pathauto config and have alias '/program', so that they exist in the breadcrumbs.
I've changed 'no_stup' option in the modules/openy_features/openy_demo_content/modules/openy_demo_nclass/config/install/migrate_plus.migration.node_class_02.yml and added 'no_stub: true' in the modules/openy_features/openy_demo_content/modules/openy_demo_nsessions/config/install/migrate_plus.migration.node_session_04.yml for 'field_session_class'. After that all migration finished successful without stub entities and issue was resolved.

@podarok, @danylevskyi please advice.
Notice: some of plugins were commented, another were updated, but in the config files of node class and node session there still exist relationship to old plugins.
Some examples in the migrate_plus.migration.node_class_02.yml:

- line 1582 - it is a typo in "activity: aactivity_arts_crafts", should be with one "a" char in the beginning;

- line 1201 -       "activity: academic_success", but academic_success plugin does not exists in the migrate_plus.migration.node_activity.yml.

- line 658 -       "activity: activity_small_group_and_specialty_training", but this plugin exists only in the migrate_plus.migration.node_activity.yml and it is commented.

These are just a few examples, in fact, there are many similar issues with plugins in both of these files.

**Comment by YMCA-GTC** [comment]
@podarok Can you please help me figure out next steps on this one? Thanks!

**Comment by ddrozdik** [comment]
@YMCA-GTC as far as I remember we fixed this in the Open Y 2.0. Let's close.

**Comment by podarok** [comment]
This is fixed in 2.0
Closing
